t/30-test_parser.t

```diff
@@ -582,6 +582,19 @@ subtest tap_parse => sub {
 
     is $_->{result}, 'ok', 'All testcases are passing' for @{$parser->results->first->details};
 
+    # test other suffix and failing test
```

Comment:
Maybe make it a subtest instead of using a comment?

Reply:
I added another tests and made them subtests now.